com.qzandronum.Q-Zandronum.yaml

```diff
@@ -0,0 +1,185 @@
+app-id: com.qzandronum.Q-Zandronum
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: 5.15-22.08
+command: doomseeker.sh
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+
+  # Based on GZDoom
+  # Now for something confusing: Doomseeker honours XDG standards
+  # But Zandronum doesn't. Both persist for consistency.
+  - --env=DOOMWADDIR=/app/share/games/doom
+  - --persist=.config/zandronum
+  - --persist=.config/doomseeker
+
+cleanup:
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+  - /lib/pkgconfig
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/glew/glew.json
+  - shared-modules/gtk2/gtk2.json
+
+  - shared-modules/SDL2/SDL2-no-libdecor.json
+  - shared-modules/SDL/sdl12-compat.json
+
+  # with libjpeg.so.8
+  - name: libjpeg
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_SKIP_RPATH:BOOL=YES
+      - -DENABLE_STATIC:BOOL=NO
+      - -DWITH_JPEG8:BOOL=YES
+      - -DCMAKE_INSTALL_LIBDIR=/app/lib
+    sources:
+      - type: archive
+        url: https://github.com/libjpeg-turbo/libjpeg-turbo/archive/refs/tags/2.1.3.tar.gz
+        sha256: dbda0c685942aa3ea908496592491e5ec8160d2cf1ec9d5fd5470e50768e7859
+
+  - name: openssl11
+    buildsystem: simple
+    build-commands:
+      - ./config --prefix=/app
+      - make -j $FLATPAK_BUILDER_N_JOBS
+      - make install
+    cleanup:
+      - /share/doc
+      - /share/man
+    sources:
+      - type: archive
+        url: https://www.openssl.org/source/openssl-1.1.1v.tar.gz
+        sha256: d6697e2871e77238460402e9362d47d18382b15ef9f246aba6c7bd780d38a6b0
+        x-checker-data:
+          type: anitya
+          project-id: 20333
+          url-template: https://www.openssl.org/source/openssl-$version.tar.gz
+
+  - name: fluidsynth1
+    buildsystem: cmake-ninja
+    disabled: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DLIB_SUFFIX=
+      - -DCMAKE_EXE_LINKER_FLAGS=-lncurse
+      - -DCMAKE_SHARED_LINKER_FLAGS=-lncurses
+    sources:
+      - type: archive
+        url: https://github.com/FluidSynth/fluidsynth/archive/refs/tags/v1.1.11.tar.gz
+        sha256: da8878ff374d12392eecf87e96bad8711b8e76a154c25a571dd8614d1af80de8
+
+  - name: gme
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-0.6.3.tar.gz
+        sha256: 626e8a104e0dadd10ef6519a67aca880c7b40f81471659f1935b61754e12fc7b
+        x-checker-data:
+          type: anitya
+          project-id: 866
+          url-template: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-$version.tar.gz
+
+  - name: bzip2
```

Comment:
it's in the the runtime. you shouldn't need to build it.

Reply:
You mean bzip2?

Reply:
Fixed.